src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -44,16 +44,16 @@
     "EmptyCall",
 )
 
-_METHOD_CAMEL_TO_CAPS_SNAKE = {
-    "UnaryCall": "UNARY_CALL",
-    "EmptyCall": "EMPTY_CALL",
-}
-
 _METHOD_STR_TO_ENUM = {
     "UnaryCall": messages_pb2.ClientConfigureRequest.UNARY_CALL,
     "EmptyCall": messages_pb2.ClientConfigureRequest.EMPTY_CALL,
 }
 
+_METHOD_CAMEL_TO_FULL_PATH = {
```

Comment:
The idea of using the full method path was reverted, for consistency with the existing fields:

https://github.com/grpc/grpc-proto/pull/90

Reply:
Switched back to using enum names.